ignite/metrics/gan/fid.py

```diff
@@ -0,0 +1,213 @@
+from typing import Callable, Optional, Sequence, Union
+
+import numpy as np
+import torch
+
+from ignite.metrics.metric import Metric, reinit__is_reduced, sync_all_reduce
+
+__all__ = ["FID", "InceptionExtractor"]
+
+
+def fid_score(
+    mu1: torch.Tensor, mu2: torch.Tensor, sigma1: torch.Tensor, sigma2: torch.Tensor, eps: float = 1e-6
+) -> Union[torch.Tensor, float]:
+
+    try:
+        import scipy
+    except ImportError:
+        raise RuntimeError("fid_score requires scipy to be installed.")
```

Comment:
@gucifer please add a test to cover this case.

Reply:
[https://github.com/gucifer/ignite/blob/49eda3b8004d9c7769896e01c83dbf76a23a1ad9/tests/ignite/metrics/gan/test_fid.py#L43](url)
The test already exists.